ui/components/app/token-list-display/token-list-display.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { isEqual } from 'lodash';
+
+import { getShouldHideZeroBalanceTokens } from '../../../selectors';
+import { useTokenTracker } from '../../../hooks/useTokenTracker';
+import Identicon from '../../ui/identicon';
+import TokenBalance from '../../ui/token-balance';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+import { getTokens } from '../../../ducks/metamask/metamask';
+
+export default function TokenListDisplay({ clickHandler }) {
+  const t = useI18nContext();
+  const shouldHideZeroBalanceTokens = useSelector(
+    getShouldHideZeroBalanceTokens,
+  );
+
+  const tokens = useSelector(getTokens, isEqual);
+  const { loading, tokensWithBalances } = useTokenTracker(
+    tokens,
+    true,
+    shouldHideZeroBalanceTokens,
+  );
+  if (loading) {
+    return <div className="loading-span">{t('loadingTokens')}</div>;
+  }
+
+  const sendableTokens = tokensWithBalances.filter((token) => !token.isERC721);
+
+  return (
+    <>
+      {sendableTokens.map((tokenData) => {
+        const { address, symbol, image } = tokenData;
+
+        return (
+          <div
+            key={address}
+            className="token-list-item"
+            onClick={() => clickHandler(tokenData)}
+          >
+            <Identicon address={address} diameter={36} image={image} />
+            <div className="token-list-item__data">
+              <div className="token-list-item__symbol">{symbol}</div>
+              <div className="token-list-item__balance">
+                <span className="token-list-item__balance__label">
```

Comment:
[req] nesting blocks isn't really valid BEM syntax for class names, if we could make this __balance-label instead that would be better. I know this is a NIT but using this forum as a way of spreading knowledge of best practices until the best practices documentation comes together.